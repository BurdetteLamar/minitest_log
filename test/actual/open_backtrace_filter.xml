<log>
  <summary verdicts='7' failures='0' errors='0'/>
  <verdict method='verdict_assert?' outcome='passed' id='default_log'>
    <act_value>
      true
    </act_value>
  </verdict>
  <verdict method='verdict_assert?' outcome='passed' id='default_ruby'>
    <act_value>
      true
    </act_value>
  </verdict>
  <verdict method='verdict_assert?' outcome='passed' id='log_log'>
    <act_value>
      true
    </act_value>
  </verdict>
  <verdict method='verdict_assert?' outcome='passed' id='log_ruby'>
    <act_value>
      true
    </act_value>
  </verdict>
  <verdict method='verdict_assert?' outcome='passed' id='ruby_log'>
    <act_value>
      true
    </act_value>
  </verdict>
  <verdict method='verdict_assert?' outcome='passed' id='ruby_ruby'>
    <act_value>
      true
    </act_value>
  </verdict>
  <verdict method='verdict_assert_equal?' outcome='passed' id='__error_count__'>
    <exp_value>
      0
    </exp_value>
    <act_value>
      0
    </act_value>
  </verdict>
</log>
