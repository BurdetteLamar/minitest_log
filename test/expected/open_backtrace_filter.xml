<log>
  <verdict_ method='verdict_assert?' outcome='passed' id='filter_minitest'>
    <actual_ class='TrueClass' value='true'/>
  </verdict_>
  <verdict_ method='verdict_assert?' outcome='passed' id='filter_xxx'>
    <actual_ class='TrueClass' value='true'/>
  </verdict_>
</log>
